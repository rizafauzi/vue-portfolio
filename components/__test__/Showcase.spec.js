import { shallowMount } from '@vue/test-utils'
import Showcase from '../Showcase.vue'

describe('Showcase.vue', () => {
	let wrapper = null

	beforeEach(() => {
		wrapper = shallowMount(Showcase)
	})
	afterEach(() => {
		wrapper.destroy()
	})

	it('should render correctly', () => {
		
		expect(wrapper.element).toMatchSnapshot()
	})

	// it('should change text label when button clicked', () => {
	// wrapper.find('.btn-action').trigger('click.prevent');

	// expect(wrapper.vm.label).toBe('Hello');
	// })
})
