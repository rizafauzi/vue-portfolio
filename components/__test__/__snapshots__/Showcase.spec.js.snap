// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Showcase.vue should render correctly 1`] = `
<div
  class="container"
>
  <div
    class="columns is-desktop is-tablet is-mobile is-multiline"
  >
    <div
      class="column"
    />
     
    <div
      class="column is-three-quarters-desktop is-full-tablet is-full-mobile"
    >
      <div
        class="tagline"
      >
        <div
          class="logo"
        >
          <div
            class="wrapper"
          >
            <div
              class="present-wrapper"
            >
              <span>
                an event by
              </span>
              <img
                alt="Narasi"
                class="narasi-logo"
                src="~/static/logo/narasi-brandmark-primary-yellow.png"
              />
            </div>
             
            <img
              alt="Playfest"
              class="playfest-logo"
              src="~/static/logo/playfest.png"
            />
             
            <div
              class="desc-mobile"
            >
              <p>
                See More , Experience Many
              </p>
            </div>
          </div>
        </div>
         
        <div
          class="desc"
        >
          <span>
            See More , Experience Many
          </span>
        </div>
      </div>
    </div>
     
    <div
      class="column is-4-desktop is-full-tablet is-full-mobile"
    >
      <div
        class="date-container"
      >
        <div
          class="date"
        >
          <img
            src="~/static/calendar.png"
          />
           
          <span>
            12 September 2020
          </span>
        </div>
      </div>
    </div>
     
    <div
      class="column is-4-desktop is-full-tablet is-full-mobile"
    >
      <div
        class="countdown-container"
      >
        <div
          class="countdown"
        >
          <div
            class="days-container"
          >
            <p
              class="days"
            >
              0
            </p>
             
            <p>
              days
            </p>
          </div>
           
          <div
            class="colon"
          >
            <p>
              :
            </p>
          </div>
           
          <div
            class="hours-container"
          >
            <p
              class="hours"
            >
              0
            </p>
             
            <p>
              hours
            </p>
          </div>
           
          <div
            class="colon"
          >
            <p>
              :
            </p>
          </div>
           
          <div
            class="minutes-container"
          >
            <p
              class="minutes"
            >
              0
            </p>
             
            <p>
              minutes
            </p>
          </div>
        </div>
      </div>
    </div>
     
    <div
      class="column is-full-desktop is-full-tablet is-full-mobile"
    >
      <div
        class="desc-container"
      >
        <div
          class="desc"
        >
          <p>
            A melting pot of great 
            <b>
              content
            </b>
            , ideas 
            <b>
              collaboration
            </b>
            , 
            <b>
              community movement
            </b>
            ,
            <br />
            and entertainment all in one experience.
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
